projects/laji/src/app/shared-modules/laji-map/visualization/laji-map-visualization.ts

```diff
@@ -0,0 +1,13 @@
+import { GetFeatureStyleOptions } from 'laji-map';
+import { MarkerCluster, PathOptions } from 'leaflet';
+
+export interface LajiMapVisualizationCategory { color: string; label?: string; translate?: string };
```

Comment:
Idk how I feel about this. Practically there probably won't be any conflicts, but just the possibility that there could be one makes me feel a bit uneasy

Reply:
I read the usage again, and I was confused because I didn't notice that there are two different `label` props (one for the visualization instance and then for the categories). So no problem here :)